config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml

```diff
@@ -310,6 +310,31 @@ spec:
                       true or leave empty.
                     type: boolean
                 type: object
+              platformType:
+                allOf:
+                - enum:
```

Comment:
@mhrivnak it seems that we can not redefine the validation list but only add on top of it. I suggest to leave it then as a custom type of PlatformType and only align the names. Is that OK? The option of a webhook validation is misleading because the CRD explicitly declares the full list as something we support.

Reply:
Yes, I think that's fine. The important part is that the string values are the same as the openshift API, and the validation works correctly.